### PR TITLE
User story 3: Hidden tasks

### PR DESCRIPTION
**Pull request by majornorth**
## Tasks
- Complete [Hide Old Tasks](https://www.bloc.io/users/stewart-mccoy/checkpoints/999?roadmap_section=8) for the Blocitoff project

### CODE REVIEW

**Comment by majornorth** [comment]
@RyanBalfanz CR? :smile:

File: app/scripts/app.js [hunk 1]
```diff
@@ -32,10 +32,19 @@ blocList.controller('Landing.controller', ['$scope', '$firebaseArray', function(
         }
         $scope.todos.$add({
             title: newTodo,
-            completed: false
+            completed: false,
+            submitted: Date.now(),
+            expiryDate: Date.now() + 604800000
```
Comment (RyanBalfanz) [inline app/scripts/app.js:37]: This is a [magic number](http://en.wikipedia.org/wiki/Magic_number_%28programming%29). Maybe consider naming it?

Comment (majornorth) [inline app/scripts/app.js:37]: haha, I love that. Yeah, I'll set it to a variable :)

**Comment by majornorth** [comment]
updated:

```
var todoDuration = 604800000;

$scope.todos.$add({
        title: newTodo,
        completed: false,
        submitted: Date.now(),
    expiryDate: Date.now() + todoDuration
});
```

**Pull request merged**